module3/assignment3.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Cole Marter | INFO 2439 | Assignment 3</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial scale=1">
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+<script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+<link rel="stylesheet" href="_css/Marter_a3.css">
+<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+<link rel="stylesheet" href="themes/Monochromatic.min.css" />
+<link rel="stylesheet" href="themes/jquery.mobile.icons.min.css" />
+<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile.structure-1.4.5.min.css" />
+<script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+<link href="https://fonts.googleapis.com/css?family=Lora&display=swap" rel="stylesheet">
+  </head>
+<body>
+<!-- Cole Marter
+     assignment3.htm
+     INFO2439
+     Thoendel
+     9/27/19
+-->
+<div data-role="page" id="page1" data-add-back-btn="true" data-title="Cole Marter | INFO 2439 | Assignment #3">
+  <div data-role="header" data-position="fixed">
+  <p>
+      <a href="" data-rel="back">Go back</a>
+      </p>
+      <h1>Assignment #3</h1>
+    </div>
+<h2>Jquery Mobile<br>Theme Roller</h2>
+  <ul data-role="listview">
+  <li><a data-ajax="false" href="ctheme1/index.html">Monochromatic Theme</a></li>
+  <li><a data-ajax="false" href="ctheme2/index.html">Complementary Theme</a></li>
```

Comment:
good use of data-ajax! When testing your assignment on the server - both ctheme1 and ctheme2 seem to be identical. So - you will lose some points there - but it looks like you've got the basic idea. Let me know if you have questions.